tests/show_techsupport/test_techsupport_no_secret.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pprint
+import pytest
+import time
+import logging
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import skip_release
+from log_messages import *
+import tech_support_cmds as cmds 
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def check_image_version(duthost):
+    """Skips this test if the SONiC image installed on DUT is older than 202112
+    Args:
+        duthost: Hostname of DUT.
+    Returns:
+        None.
+    """
+    skip_release(duthost, ["201811", "201911", "202012", "202106"])
+
+@pytest.fixture
+def setup_password(duthosts, enum_rand_one_per_hwsku_hostname, creds_all_duts):
```

Comment:
>setup_password [](http://example.com/codeflow?start=4&length=14)

Duplicated code. Could you reuse?

Reply:
Fixed, remove dupe define of setup_password from test_techsupport.py